README.md

```diff
@@ -43,12 +43,14 @@ Options:
 Get help for individual commands by running fliegdoc <command> --help
 ```
 
-The CLI searches for a `fliegdoc.config.js` file and applies its options on top of the default options.
+The CLI searches for a `fliegdoc.config.js` file and applies its options **on top of the default options**.
 
 #### Example `fliegdoc.config.js` with default options
 
 ```js
 // fliegdoc.config.js
+const { HTMLTheme } = require('fliegdoc');
```

Comment:
`require` is not possible in `.yml` and similar files.
Can you use a string like `fliegdoc-html-parser` instead and require it in the configuration builder?
You can also disable support for `.yml` and similar files.
See: https://www.npmjs.com/package/cosmiconfig#searchplaces

Reply:
For now, I don't really want to support strings as this means I'd have to gather the theme's location, and if it has a valid structure, a proper export, and so on.

Since no-where, I write that it also supports `.yml` files, personally, I'd "silently" support `.yml` configuration (after all, in a lot of cases, specifying a different theme isn't necessary) 🤔 .

Reply:
I have created a discussion for this as a feature request: https://github.com/fliegwerk/fliegdoc/discussions/9

Reply:
Okay, so you need in https://github.com/fliegwerk/fliegdoc/blob/ebd3331e7a50ab1163f89c31b75db32c5dbe066b/bin/fliegdoc.js#L19
something like:
```js
const moduleName = 'fliegdoc';
const { config, filepath } = cosmiconfigSync(moduleName, {
	searchPlaces: [
		`.${moduleName}rc.js`,
		`.${moduleName}rc.cjs`,
		`${moduleName}.config.js`,
		`${moduleName}.config.cjs`
	]
}).search();